docs/manual-testing.md

```diff
@@ -0,0 +1,39 @@
+Martha in Terra     Manual End-to-End Test
```

Comment:
I am worried that having these steps copied over into this file means we have multiple places where we need to maintain this test definition

Reply:
I'm not sure what the correct answer is here.  Maybe it's based on why your team felt they wanted the test steps copied over?  Perhaps the solution is to have a version controlled test definition instead of a Google Doc?

Reply:
I would definitely rather have these steps (1) all in the same place and (2) versioned.